templates/node.js-express-webapp-on-linux.yml

```diff
@@ -6,35 +6,77 @@
 trigger:
 - master
 
-pool:
-  vmImage: 'ubuntu-latest'
-
-steps:
-- task: NodeTool@0
-  inputs:
-    versionSpec: '10.x'
-  displayName: 'Install Node.js'
-
-- script: |
-    npm install
-    npm build
-  displayName: 'npm install and build'
-
-- task: ArchiveFiles@2
-  displayName: 'Archive files'
-  inputs:
-    rootFolderOrFile: '$(System.DefaultWorkingDirectory)'
-    includeRootFolder: false
-    archiveType: zip
-    archiveFile: $(Build.ArtifactStagingDirectory)/$(Build.BuildId).zip
-    replaceExistingArchive: true
-    
-- task: AzureWebApp@1
-  displayName: 'Azure Web App Deploy: expressappyaml'
-  inputs:
-    azureSubscription: '{{ azureRmConnection.Name }}'
-    appType: webAppLinux
-    appName: '{{ webAppName }}'
-    runtimeStack: 'NODE|10.10'
-    package: $(Build.ArtifactStagingDirectory)/$(Build.BuildId).zip
-    startUpCommand: 'npm run start'
\ No newline at end of file
+variables:
+
+  # Azure Resource Manager connection created during pipeline creation
+  azureSubscription: '{{ azureRmConnection.Name }}'
+  
+  # Web app name
+  webAppName: '{{ webAppName }}'
+
+  # Agent VM image name
+  vmImageName: 'ubuntu-latest'
+
+stages:
+- stage: Build
+  displayName: Build stage
+  jobs:  
+  - job: Build
+    displayName: Build job

```

Comment:
We don't need `job` suffix for the displayName. Currently it shows up as 'Build job job'

Reply:
Same for deploy job

Reply:
Done